File: evaluation/suites/baseline-v1/fixtures/python-mini/inventory.py
class Inventory:
    def __init__(self) -> None:
        self._items: dict[str, int] = {}

    def add(self, sku: str, quantity: int) -> None:
        self._items[sku] = self._items.get(sku, 0) + quantity

    def quantity(self, sku: str) -> int:
        return self._items.get(sku, 0)

File: penhin/evaluation/suites/baseline-v1/fixtures/javascript-mini/src/inventory.js
export class Inventory {
  constructor() {
    this.stock = new Map();
  }

  add(sku, quantity) {
    if (quantity < 0) throw new RangeError("quantity must be non-negative");
    this.stock.set(sku, (this.stock.get(sku) ?? 0) + quantity);
  }

  quantity(sku) {
    return this.stock.get(sku) ?? 0;
  }
}
